### PR TITLE
Change logic for unsupported/not current Player version

### PR DESCRIPTION
**Pull request by fjvallarino**
@alex-deaconu please review

### CODE REVIEW

File: web/scripts/displays/services/svc-display.js [hunk 1]
```diff
@@ -19,11 +19,19 @@
...
-        getProductSubscriptionStatus, DISPLAY_WRITABLE_FIELDS, DISPLAY_SEARCH_FIELDS, PLAYER_PRO_PRODUCT_CODE) {
+        getProductSubscriptionStatus, getLatestPlayerVersion, DISPLAY_WRITABLE_FIELDS,
+        DISPLAY_SEARCH_FIELDS, PLAYER_PRO_PRODUCT_CODE) {
+
+        var latestPlayerVersion = null;
+
+        getLatestPlayerVersion()
+        .then(function(date) {
+          latestPlayerVersion = date;
+        });
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-display.js]: @fjvallarino is this piece of code necessary? I don't see this variable being used in this service. Thanks.

Comment (fjvallarino) [reply web/scripts/displays/services/svc-display.js]: @alex-deaconu removed. This logic had been refactored and this part was not removed in time.

File: web/scripts/displays/services/svc-display-factory.js [hunk 2]
```diff
@@ -49,9 +63,17 @@ angular.module('risevision.displays.services')
       };
 
       factory.isOutdatedPlayer = function (display) {
-        return !factory.is3rdPartyPlayer(display) && (display && display.playerName && (display.playerName !==
-          'RisePlayerElectron' ||
-          display.playerVersion < '2017.07.17.20.21'));
+        var MONTH_IN_MS = 30 * 24 * 60 * 60 * 1000;
+        var displayPlayerVersion = display && parsePlayerDate(display.playerVersion);
+        var minimumVersion = _latestPlayerVersion && new Date(_latestPlayerVersion.getTime() - MONTH_IN_MS);
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-display-factory.js]: @fjvallarino why not use something like `minimumVersion = (new Date()).setMonth(_latestPlayerVersion.getMonth() - 1)` instead. This way you avoid the messy millisecond calculations. Just a suggestion. Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino looks good 👍 . Thanks for the fixes!

**Pull request merged**